src/browser/components/MainPage.jsx

```diff
@@ -62,6 +58,27 @@ export default class MainPage extends React.Component {
     this.markReadAtActive = this.markReadAtActive.bind(this);
   }
 
+  parseDeeplinkURL(deeplink, teams = this.props.teams) {
+    if (deeplink && Array.isArray(teams) && teams.length) {
+      const deeplinkURL = url.parse(deeplink);
+      let parsedDeeplink = null;
+      teams.forEach((team, index) => {
+        const teamURL = url.parse(team.url);
+        if (deeplinkURL.host === teamURL.host) {
+          parsedDeeplink = {
+            teamURL,
+            teamIndex: index,
+            originalURL: deeplinkURL,
+            url: `${teamURL.protocol}//${teamURL.host}${deeplinkURL.pathname}`,
```

Comment:
Does "team" here actually refer to a server tab? I feel like this naming might date back to before a single user could be on multiple teams

Reply:
Yes, good point. There was some work done in another PR to change all `team` references to `server` where actually server, but a bunch of work had been done since making that PR almost impossible to merge. This still needs to be done, but I think it should be done as a whole instead piece by piece to avoid cross-over confusion.